src/Facet.php

```diff
@@ -206,7 +214,7 @@ private function fetchItems($name = NULL, $dateGranularity = 'year') {
         $this->active = FALSE;
         $items = array();
         foreach ($keyedByName as $itemName => $itemCount) {
-            if (!empty($itemCount)) {
+            if (!empty($itemCount) && $itemCount > $miniCount) {
```

Comment:
@jhuang0427 I think if we're calling this a "minimum facet count" then this should probably be >= instead of >, right? In other words if the minimum is 5, then a facet count of exactly 5 should be displayed.

Reply:
Sorry I did not aware this situation.